arkouda/pdarraycreation.py

```diff
@@ -1009,13 +1009,13 @@ def random_strings_lognormal(
 
     repMsg = generic_msg(
         cmd="randomStrings",
-        args="{} {} {} {} {} {}".format(
-            NUMBER_FORMAT_STRINGS["int64"].format(size),
-            "lognormal",
-            characters,
-            NUMBER_FORMAT_STRINGS["float64"].format(logmean),
-            NUMBER_FORMAT_STRINGS["float64"].format(logstd),
-            seed,
-        ),
+        args={
+            "size": NUMBER_FORMAT_STRINGS["int64"].format(size),
+            "dist": "lognormal",
+            "chars": characters,
+            "min": NUMBER_FORMAT_STRINGS["float64"].format(logmean),
+            "max": NUMBER_FORMAT_STRINGS["float64"].format(logstd),
```

Comment:
it's unfortunate that these arguments mean different things depending on the distribution, maybe we could name them `min/mean` and `max/std`? Or just generic `argstr1/2` like they were previously in the chapel code? I'm also fine with them as is

Reply:
True, I will update it. I am leaning towards the more generic `arg1`, `arg2`.

Reply:
Updated from `min` and `max` to `arg1` and `arg2`.